fake-api.js

```diff
@@ -37,7 +39,7 @@ function fakeApiMiddleWare (req, res, next) {
     // respond with 200
     res.sendStatus(200).end();
     return;
-  } else {
+  } else if (publicEndpoints.indexOf(req.path) < 0){
```

Comment:
I don't quite understand the reason for this change. It seems you're disabling the authorization check for the request to the `/version` endpoint on the fake API, but I'm not sure why?

Reply:
I'm disabling the auth check in the `fake-api` for `/version` because it's not required to be authenticated agains the `default` api. As such, I'm not setting an `Authorization` header in the request from the dashboard to the `/version` endpoint